AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -154,7 +176,10 @@ private extension SearchMapView {
                         .onMarkerChange(action: { pointer in
                             viewModel.markerSelectionChanged(using: pointer)
                         })
-                        .border((viewModel.cardPointerID.number == session.id ? Color.accentColor : .clear), width: 1)
+                        .overlay(
+                            RoundedRectangle(cornerRadius: 8)
+                                .stroke((viewModel.cardPointerID.number == session.id ? Color.accentColor : .clear), lineWidth: 1)
+                        ).padding([.all], 2)
```

Comment:
A small thing, but you can use just `.padding(2)`

Reply:
oh, thanks for that! A lot cleaner